src/utils.ts

```diff
@@ -397,6 +397,7 @@ export function isEntryAContructor(
             line = line.trim();
             if (line.startsWith("//")) {
                 // Ignore single-line comment.
+                index++;
                 continue;
```

Comment:
Since we haven't merged yet, I'll take the opportunity to suggest that instead of doing
```typescript
index++;
continue;
```
we simply do nothing. I.e. the `if` body should just be the comment `// Ignore single-line comment`, and the `index` increment will happen regardless at the end of the loop body.